obspy/imaging/scripts/scan.py

```diff
@@ -73,23 +73,23 @@ def compressStartend(x, stop_iteration):
 
 
 def parse_file_to_dict(data_dict, samp_int_dict, file, counter, format=None,
-                       verbose=False, ignore_links=False):
+                       verbose=False, quiet=False, ignore_links=False):
     from matplotlib.dates import date2num
     if ignore_links and os.path.islink(file):
-        print("Ignoring symlink: %s" % (file))
+        if verbose or not quiet:
```

Comment:
No. By default, both `verbose` and `quiet` are False. Thus the conditional evaluates to True and the default behavior is retained. The `verbose` flag furthermore overwrites the `quiet` flag.